views/partials/form.handlebars

```diff
@@ -1 +1,42 @@
-Edit topic form
\ No newline at end of file
+Edit topic form
+
+<div class="row">
+    <p>Create a new Blog post</p>
+    <form action="/save-topic" method="POST" name="sentMessage" id="contactForm" novalidate>
+        <div class="row control-group">
+            <div class="form-group col-xs-12 floating-label-form-group controls">
+                <label>After Class</label>
+                <input name="afterClass" type="text" class="form-control" placeholder="Blog afterClass" id="afterClass" required data-validation-required-message="Please enter the title of the blog post.">
+                <p class="help-block text-danger"></p>
+            </div>
+        </div>
+        <div class="row control-group">
+            <div class="form-group col-xs-12 floating-label-form-group controls">
+                <label>monday</label>
+                <textarea name="monday"rows="5" class="form-control" placeholder="monday" id="monday" required data-validation-required-message="Please enter the summary of the blog post."></textarea>
```

Comment:
This is great, thanks for changing to "name" attribute to point to topic properties (for example `name='monday'`!

Could we change this from a `textarea` element to an `<input type='text'>` please? Text areas are usually good for larger pieces of writing (for example a blog post, or a contact us form, where you want users to write a message). While here, this input will only contain one word, so a simple `<input type='text'>` might be more suitable.